src/iop/colorchecker.c

```diff
@@ -1327,31 +1327,31 @@ void gui_init(struct dt_iop_module_t *self)
   dt_color_picker_new(self, DT_COLOR_PICKER_POINT_AREA, g->combobox_patch);
 
   g->scale_L = dt_bauhaus_slider_new_with_range(self, -100.0, 200.0, 1.0, 0.0f, 2);
-  gtk_widget_set_tooltip_text(g->scale_L, _("lightness offset"));
+  gtk_widget_set_tooltip_text(g->scale_L, _("scale lightness in Lab space\nlower values darken target color while higher brighten it"));
```

Comment:
To be consistent with below wording:

`scale target in Lab L dimension\n...`

?